swagger/v1/api_spec.json

```diff
@@ -3469,7 +3527,7 @@
           "type": {
             "description": "The data type",
             "type": "string",
-            "example": "participant-outcome"
+            "example": "npq-outcome-confirmation"
```

Comment:
This doesn't look right for this endpoint

Reply:
Hi @peteglondon it's inline with the acceptance criteria of the card.. and with the [documentation](https://manage-training-for-early-career-teachers.education.gov.uk/api-reference/reference-v3.html#schema-npqoutcomerequest) but yeah I agree, maybe it can be reviewed for a better description.